aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -83,6 +106,15 @@
       "examples": [
         "arn:aws:codeguru-profiler:us-east-1:000000000000:profilingGroup/My-example-profiling-group"
       ]
+    },
+    "Tags": {
+      "description": "The tags associated with a profiling group.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Tag"
+      },
+      "maxItems": 50,
+      "uniqueItems": false
```

Comment:
What reasons do we have for setting *uniqueItems* false or true? On first look, I would be inclined for *true*. I see some other resources have both true and false: [link for GitHub search](https://github.com/search?l=JSON&q=org%3Aaws-cloudformation+tags+uniqueItems&type=Code). What do you think?

Reply:
Good point. I did think of making it true but then I looked into ElasticSearch implementation of Tags and they have set it to false. I dont think it would cause us much trouble to be fair but I agree that true makes more sense. 

I would change it in next revision